lucille-core/src/main/java/com/kmwllc/lucille/core/SingleBatch.java

```diff
@@ -0,0 +1,93 @@
+package com.kmwllc.lucille.core;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.LinkedBlockingQueue;
+
+/**
```

Comment:
This class-level javadoc and all the method-level javadocs should be copied or moved to the Batch interface.

Reply:
Thanks @rseitz I moved them.